integration/test_redis_specific.py

```
@@ -25,46 +27,69 @@ def redis():
     return get_redis_client()
 
 def test_redis_driver_write(redis):
-    driver = RedisDriver(redis)
-    controller = build_flow([
-        SyncEmitSource(),
-        NoSqlTarget(Table('test', driver)),
-        Complete()
-    ]).run()
-    controller.emit({'col1': 0}, 'key').await_result()
-    controller.terminate()
-    controller.await_termination()
+    try:
+        table_name = 'test_redis_driver_write'
 
-    data = driver.redis.hgetall("storey:test/key:static")
-    data_strings = {}
-    for key, val in data.items():
-        if isinstance(key,bytes):
-            data_strings[key.decode('utf-8')] = val.decode('utf-8')
-        else:
-            data_strings[key] = val
+        driver = RedisDriver(redis)
+        controller = build_flow([
+            SyncEmitSource(),
+            NoSqlTarget(Table(table_name, driver)),
+            Complete()
+        ]).run()
+        controller.emit({'col1': 0}, 'key').await_result()
+        controller.terminate()
+        controller.await_termination()
+        
+        table_name = f'{table_name}/'
+        hash_key = RedisDriver.make_key('storey:', table_name, 'key')
+        redis_key= RedisDriver._static_data_key(hash_key)
 
-    assert data_strings == {"col1": '0'}
+        data = driver.redis.hgetall(redis_key)
+        data_strings = {}
+        for key, val in data.items():
+            if isinstance(key,bytes):
+                data_strings[key.decode('utf-8')] = val.decode('utf-8')
+            else:
+                data_strings[key] = val
 
+        assert data_strings == {"col1": '0'}
+    except:
+            pass
```

Why are we ignoring all errors?

---

right...should be raise :)